index.php

```diff
@@ -4,29 +4,53 @@
 require_once 'src/SeedsViewer.php';
 require_once 'src/Seed.php';
 
+
+
 $db = new PDO('mysql:host=db; dbname=seed_vault', 'root', 'password');
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC); 
 
+$query = $db->prepare('SELECT `seeds`.`id`, `seeds`.`name`, `plant_family`.`common_name` AS "Plant Family",
+`seeds`.`species`, `seeds`. `image`, `seeds`.`description`
+FROM `seeds`
+	INNER JOIN `plant_family`
+		ON `seeds`.`genus` = `plant_family`.`id`');
+
+$query->execute();
+
+$result = $query->fetchAll();
+
```

Comment:
I don't think you need this - you already have `$seeds = $seedsModel->getAllSeeds();  ` below that should be doing the same job